app.js

```diff
@@ -0,0 +1,17 @@
+const form = document.querySelector('form');
+const input = document.querySelector('input');
+const ul = document.createElement('ul');
+const li = document.createElement('li');
+
+addTask = (e) => {
```

Comment:
This is really bad practice, you should adjust type of this variable.
`const addTask  (e) => {}`;